aces/Loris
# [battery_manager] Clarify Permission Labels

## Brief summary of changes

#### Testing instructions (if applicable)

1. Run to patch.
2. Go to User accounts and make sure that the Permission labels match the labels in the patch.

#### Link(s) to related issue(s)

* Resolves #6683
